### ISSUE TITLE
Ollama Model Selection System

### ISSUE THREAD

**Issue by Someguitarist** [brunostjohn/perplexideez#10]
First, I love this so much.  This is probably my favorite github project I've seen in a while.  I've been using websearch on Open Web UI but this is so much better/faster/effective that it's crazy.  Great job!  Some minor things; 

**Is your feature request related to a problem? Please describe.**
There should be an 'Admin' webpage that lets you select your desired model for each quality that users see, rather than editing a .env file.  The program also shouldn't crash if a model isn't found, it should just let you know that that quality selection is unavailable.

**Describe the solution you'd like**
An 'Admin' web view where you could change/select models

**Additional context**
It also shouldn't crash if a model isn't available.  Took me a while to find out a K_L model I was using was actually a K_M and that's why I couldn't get it this to launch.

**Comment by brunostjohn** [comment]
An admin panel was on my to-do list for this. I didn't consider this feature for it, but it actually sound quite good! I'll keep track of this. I can't promise I'll get it done in the next release, but it'll either be this one or the next. Thank you!

**Comment by Someguitarist** [comment]
Yeah, this can definitely wait compared to everything else.  Just figured it's worth a mention!
